discovery-provider/scripts/start.sh

```diff
@@ -1,12 +1,18 @@
 #!/bin/bash
 set -e
 
-mkdir -p /var/log
-mkdir -p /var/spool/rsyslog
-mkdir -p /etc/rsyslog.d
-
-if [[ -z "$audius_loggly_disable" ]]; then
-    if [[ -n "$audius_loggly_token" ]]; then
+# enable rsyslog if not set to 0
+: "${enableRsyslog:=1}"
```

Comment:
Going forward we shouldn't use negative-based envars like `$logglyDisable` and instead use positive-based envars like `$enableLoggly` to reduce double-negative logic in favor of readability and simplicity.

To remove double negative logic like `-z $enableLoggly` which tests if `$enableLoggly` is an empty/null, we should use the `((enableLoggly))` notation which tests if `$enableLoggly` is greater than `0`, for example.

Additionally, we can't use `-n`, which tests if `$enableLoggly` is a nonzero-length string, in combination with envars set to default values like `"${enableLoggly:=1}"` since an empty `$enableLoggly` would always resolve to `1`.

For the above reasons, I've introduced and used `$enableRsyslog` in a manner different than current start.sh logic.